fix(style guide): Rewrite homepage doc (#11063)
newrelic/docs-website: src/content/docs/style-guide/writing-docs/processes-procedures/edit-homepage.mdx

  - /docs/style-guide/processes-procedures/rename-or-redirect-document
---

> You can't just hit the edit button docs.newrelic.com to make edits to the home page. The homepage copy and code is governed by two pages:

Reviewer: ```suggestion
You can't just hit the edit button on docs.newrelic.com to make edits to the home page. The homepage copy and code is governed by two pages:
```

* [`src/pages/index.js`](https://github.com/newrelic/docs-website/blob/develop/src/pages/index.js): Tile structure, link destinations, and logic.
* [`src/i18n/translations/en/translation.json`](https://github.com/newrelic/docs-website/blob/develop/src/i18n/translations/en/translation.json): Titles for tiles and some other homepage copy.

> 

Reviewer: ```suggestion
See the sections below to make specific changes to the home page. 
```

## Update a link's URL

> To change or add a link, you'll need to update the relevant copy in both `index.js` and `translation.json`. The exact syntax depends on which part of the homepage you're changing. The easiest way to make the update is to search each file for the relevant text, make your change, and check the preview build.

Reviewer: ```suggestion
To change or add a link, you'll need to update the relevant copy in both `index.js` and `translation.json`. The exact syntax depends on which part of the home page you're changing. The easiest way to make the update is to search each file for the relevant text, make your change, and check the preview build.
```
